openapi/components/schemas/QuoteChangeOrder.yaml

```
@@ -92,147 +78,30 @@ properties:
-                a customer may receive a notification
-                but the service can still be provided up to the hard limit value so that the customer can upgrade their plan.
-                When the reported usage reaches the configured limit,
-                a specific event is triggered.
-                To notify the customer in the merchant system,
-                or block a service,
-                a webhook and notification can be configured for this event.
-                When the total usage reaches the hard limit quantity,
-                or amount values,
-                metered billing plan usages can no longer be reported.
-              minProperties: 1
-              maxProperties: 1
-              properties:
-                quantity:
-                  type: integer
-                  description: Usage limit quantity.
-                  minimum: 1
-                amount:
-                  type: number
-                  format: double
-                  description: Usage limit amount in the currency of the order.
-                  minimum: 0.01
-        createdTime:
-          $ref: ./CreatedTime.yaml
-        updatedTime:
-          $ref: ./UpdatedTime.yaml
-        _embedded:
-          type: object
-          description: Embedded objects that are requested by the `expand` query parameter.
-          readOnly: true
-          properties:
-            product:
-              type: object
-  renewalPolicy:
-    description: Specifies if the subscription retains its current `renewalTime` or resets it to a newly calculated `renewalTime`.
+  orderId:
+    description: ID of the related order.
     type: string
-    enum:
-      - reset
-      - retain
-  prorated:
-    description: |-
-      Specifies whether to give a pro rata credit for the amount of time remaining between the `effectiveTime` and the end of the current period.
-
-      In addition, if the `renewalTime` is retained, by setting the `renewalPolicy` to `retain`,
-      a pro rata debit occurs for the amount between the `effectiveTime` and the `renewalTime` as a percentage of the normal period length.
-    type: boolean
-  effectiveTime:
-    description: |-
-      Date from which the renewal time for `reset` operations and proration calculations are made.
-      If this field is omitted, this value defaults to the time of quote acceptance.
-    type:
-      - 'string'
-      - 'null'
-    format: date-time
-  keepTrial:
-    description: |-
-      Specifies if the subscription order must retain its active trial.
-      This field is only applicable if `renewalPolicy` is set to `retain`.
-    type: boolean
-    default: false
-  interimOnly:
-    type: boolean
-    default: true
-    description: |-
-      Specifies if the quotation invoice must include interim items only.
-      If this value is set to `false`, all upcoming items are included.
+    maxLength: 50
+    example: ord_0YV7DES3WPC5J8JD8QTVNZBZNZ
```

To unify the same prefix example for each resource.

```suggestion
    example: ord_01HVKA5975PJBSQ1SX72G3MSZC
```